go.mod

```diff
@@ -4,7 +4,7 @@ go 1.19
 
 require (
 	github.com/google/go-cmp v0.5.9
-	github.com/hashicorp/terraform-plugin-go v0.18.0
+	github.com/hashicorp/terraform-plugin-go v0.18.1-0.20230824194237-31d190886564
```

Comment:
This will be updated to terraform-plugin-go v0.19.0 pending https://github.com/hashicorp/terraform-plugin-go/pull/311 review, merge, and release.